$NetBSD: patch-ab,v 1.3 2006/07/28 12:26:18 rillig Exp $

We don't need the dependencies for building the package once.
Additionally, not every compiler understands the -MG option.

--- rules.mk.in.orig	2005-07-03 02:13:51.000000000 +0200
+++ rules.mk.in	2006-07-28 13:19:38.529610934 +0200
@@ -141,9 +141,6 @@ Makefile.depends: $(NEOTONIC_ROOT)/rules
 	@echo "** OSNAME: $(OSTYPE)"
 	@rm -f Makefile.depends
 	@touch Makefile.depends
-	@for II in `find . -maxdepth 1 -name "*.c" -print`; do \
-		gcc -M -MG ${CFLAGS} $$II >> Makefile.depends; \
-	done;
 	@echo "** (done) "
 
 DEPEND_FILE := $(shell find . -name Makefile.depends -print)
